packages/api/src/webhooks/cannon.ts

```diff
@@ -290,43 +375,102 @@ export default class WebhookCannon {
       // sign payload if there is a webhook secret
       if (webhook.sharedSecret) {
         let signature = sign(params.body, webhook.sharedSecret);
-        params.headers["Livepeer-Signature"] = `t=${timestamp},v1=${signature}`;
+        params.headers[SIGNATURE_HEADER] = `t=${timestamp},v1=${signature}`;
+      }
+      try {
+        await this.storeTriggerTime(webhook);
```

Comment:
Hmm, I think we should make sure to use the same timestamps for both the `lastTriggerTime` and the `lastFailure` fields, otherwise we would never be able to tell whether the last failure was actually the last trigger or not. Maybe declare the field locally here and send it as an argument to both functions.